### PR TITLE
Bare minimum kafka source e2e tests

### PR DESCRIPTION
**Pull request by lionelvillard**
# Changes

- Add e2e tests plumbing. Add one test doing nothing else than testing KafkaSource readiness (which should move to `eventing-kafka`)
-
-

/kind enhancement

Fixes #

**Release Note**

```release-note
```

**Docs**

```docs

```

### CODE REVIEW

**Comment by lionelvillard** [comment]
/hold

it's still using an old API

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+[`e2e-tests.sh`](e2e-tests.sh) is the entry point for running all e2e tests.
+
+You can run it simply with:
+
+```shell
+test/e2e-tests.sh (--kafka-source|--kafka-mt-source)
+```
+
+_By default, it will create a new GKE cluster in project `$PROJECT_ID`, start
```
Comment (zroubalik) [inline test/README.md:48]: Does it start KEDA as well?

**Review by zroubalik — COMMENTED** [review]
This is great, have a question on unit tests mentioned in the Readme. What are those about?

File: test/README.md (hunk 1 above)
Comment (lionelvillard) [reply test/README.md:48]: yes

Comment (zroubalik) [reply test/README.md:48]: Might be worth adding that there

File: test/README.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+Knative Serving and Eventing system, upload test images to `$KO_DOCKER_REPO`,
```
Comment (zroubalik) [inline test/README.md:49]:
```suggestion
KEDA, Knative Serving and Eventing system, upload test images to `$KO_DOCKER_REPO`,
```

File: test/README.md (hunk 1 above)
Comment (lionelvillard) [reply test/README.md:48]: Added. Actually the bash function was there but not called. I also fixed that

**Comment by lionelvillard** [comment]
I edited the README. KEDA is now installed.

I also removed `config-logging` from the controller yaml, as this shouldn't be needed. @zroubalik Can you confirm?

**Comment by lionelvillard** [comment]
> This is great, have a question on unit tests mentioned in the Readme. What are those about?

These are the regular Go unit tests. There is not much today except the one testing the Kafka feature.

**Review by zroubalik — APPROVED** [review]

**Pull request merged**